services/ir1/artifacts.mk

```diff
@@ -1,13 +1,13 @@
 # Get NeoFS IR artifacts (NeoFS contracts, ADM and CLI)
 
-get.ir: get.cli get.contracts get.adm get.storage prepare.storage
+get.ir1: get.cli get.contracts get.adm get.storage prepare.storage
 
 # Download NeoFS CLI 
 .ONESHELL:
 get.cli: NEOFS_CLI_FILE=./vendor/neofs-cli
 get.cli: NEOFS_CLI_PATH?=
 get.cli:
-	@touch services/ir/.ir.env # https://github.com/docker/compose/issues/3560
+	@touch services/ir${IR_NUMBER_OF_NODES}/.ir.env # https://github.com/docker/compose/issues/3560
```

Comment:
also placed in `.../ir1/...` dir but looks like it should support different numbers too